inst/components/scss/page_sidebar.scss

```
@@ -44,15 +44,15 @@ $bslib-sidebar-padding: $spacer * 1.5 !default;
   // Ensure the page-level main area has a minimum height and width to prevent
   // overly squished content in small screens, like IDE preview panels.
   .bslib-sidebar-layout {
-    .bslib-page-main {
+    .bslib-page-main.html-fill-container {
```

Since the `min-height` here is more about the `flex-shrink` property set `.html-fill-item`, this selector would be a bit more obvious to me

```suggestion
    .bslib-page-main.html-fill-item {
```